src/index.js

```diff
@@ -125,10 +160,12 @@ function startSearch(searchTerm) {
     .then(mvnSearchResponseArrived);
 }
 
-export function search(argv) {
-    if (argv.length < 3 || argv[2].trim() === "") {
-    newSearch();
+export function search(term, format) {
+    searchTerm = term || '';
```

Comment:
If `term` is empty, then we should call `newSearch()` (at least that was the previous behavior - is there any reason for dropping that?).

Reply:
line 166 still contains this behavior! I did not change that behavior. When just calling `mvn-search` it prompts for a term. :)